rustic-rustfmt.el

```diff
@@ -188,6 +196,38 @@ and it's `cdr' is a list of arguments."
         (kill-buffer proc-buffer)
         (message "Workspace formatted with cargo-fmt.")))))
 
+;;;###autoload
+(defun rustic-format-region (begin end)
+  "Format the current active region using rustfmt.
+
+This operation requires a nightly version of rustfmt.
+"
+  (interactive "r")
+  (unless (or (eq major-mode 'rustic-mode)
+              (eq major-mode 'rustic-macro-expansion-mode))
+    (error "Not a rustic-mode buffer."))
+  (let* ((buf (current-buffer))
+         (file (buffer-file-name buf))
+         (r-begin)
+         (r-end))
+    (cond ((region-active-p)
```

Comment:
Maybe we should just do something like if region is active -> format region, if not format buffer ?

Reply:
Sounds good